app/Commands/Php.php

```diff
@@ -41,6 +42,6 @@ public function handle(Process $process)
      */
     public function getDefaultArgs(): string
     {
-        return 'php -v';
+        return '-v';
```

Comment:
shouldn't this be moved to the builder as well? not sure we kept both defaultArgs working?

Reply:
having both working is good enough for now. Let's keep in mind to pretty soon move them all.